packages/lambda-powertools-middleware-correlation-ids/event-sources/alb.js

```diff
@@ -0,0 +1,50 @@
+const CorrelationIds = require('@dazn/lambda-powertools-correlation-ids')
+const Log = require('@dazn/lambda-powertools-logger')
+const consts = require('../consts')
+
+function isMatch (event) {
+  return event.hasOwnProperty('httpMethod') && event.requestContext.hasOwnProperty('elb')
+}
+
+function captureCorrelationIds ({ requestContext, headers }, { awsRequestId }, sampleDebugLogRate) {
+  if (!headers) {
+    Log.warn(`Request ${awsRequestId} is missing headers`)
+    return
+  }
+
+  const albRequestId = awsRequestId
+  const correlationIds = { awsRequestId, albRequestId }
```

Comment:
does the ALB not pass its own request ID through like API GW? if that's the case we probably don't need a separate `albRequestId`

Reply:
On APIGW, you get a lot of information in the request context - details about the API Gateway resources involved in the request, as well as other features of APIGW. On the other hand, the ALB object is very simple giving you only the ARN of the target group that is being invoked.